pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -3113,6 +3113,21 @@ let
         };
       };
 
+      ms-vscode.test-adapter-converter = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "test-adapter-converter";
+          publisher = "ms-vscode";
+          version = "0.1.9";
+          sha256 = "sha256-M53jhAVawk2yCeSrLkWrUit3xbDc0zgCK2snbK+BaSs=";
+        };
+        meta = {
+          description = "A Visual Studio Code extension that converts from the Test Explorer UI API into native VS Code testing. ";
```

```suggestion
          description = "A Visual Studio Code extension that converts from the Test Explorer UI API into native VS Code testing";
```
It should end without `.`. Also no spaces.